common/src/main/java/org/apache/celeborn/common/meta/FileInfo.java

```diff
@@ -265,4 +277,33 @@ public boolean isPartitionSplitEnabled() {
   public void setPartitionSplitEnabled(boolean partitionSplitEnabled) {
     this.partitionSplitEnabled = partitionSplitEnabled;
   }
+
+  public void setSorted() {
+    synchronized (sorted) {
```

Comment:
This syncrhonize block is redundant.

Reply:
it's for blocking the `addStream`

Reply:
I agree with @FMX , the synchronization in `addStream` and `isStreamsEmpty` are necessary because these two methods need to be sequential, but it's unnecessary to synchronize in `setSorted` and `closeStream`.